### PR TITLE
Add Footer

### LINKED ISSUE #3: Footer
Screenshot 2023-11-07 at 14 52 50

### PR DESCRIPTION
**Pull request by bosschaert**
Fixes #3

Test URLs:
- Original: https://www.sunstar-foundation.org/en/
- Before: https://main--sunstar-foundation--hlxsites.hlx.live/en/
- After: https://issue-3--sunstar-foundation--hlxsites.hlx.live/en/

### CODE REVIEW

File: blocks/footer/footer.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ export default async function decorate(block) {
 
   // fetch footer content
   const footerMeta = getMetadata('footer');
-  const footerPath = footerMeta || (getLanguage() === 'en' ? '/footer' : `/${getLanguage()}/footer`);
+  const footerPath = footerMeta || (getLanguage() === 'jp' ? '/footer' : `/${getLanguage()}/footer`);
```
Comment (dnbute) [inline blocks/footer/footer.js:57]: Same as for the footer, the getLanguage() logic needs updating

**Review by dnbute — COMMENTED** [review]
Screenshot 2023-11-14 at 10 34 27
Our footer is bigger (might be different for mobile as well but it's kinda hard to check side-by-side), same as the nav, I think it's fine but just letting you know

**Review by dnbute — APPROVED** [review]

File: blocks/footer/footer.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ export default async function decorate(block) {
 
   // fetch footer content
   const footerMeta = getMetadata('footer');
-  const footerPath = footerMeta || (getLanguage() === 'en' ? '/footer' : `/${getLanguage()}/footer`);
+  const footerPath = footerMeta || (getLanguage() === 'ja' ? '/footer' : `/${getLanguage()}/footer`);
```
[the remark this answers is not in the document]
Comment (bosschaert) [reply blocks/footer/footer.js:57]: For this website the default language (at the root) is Japanese. The English site is under https://hostname/en/...

**Comment by bosschaert** [comment]
As the differences found are expected and non-disruptive I will merge this PR now. If we need additional changes we can always make them later.

**Pull request merged**